### ISSUE TITLE
examples/gaussian_mixture_model.ipynb (update var to vars)

### ISSUE THREAD

**Issue by x2apps** [pymc-devs/pymc#956]
https://github.com/pymc-devs/pymc3/blob/master/pymc3/examples/gaussian_mixture_model.ipynb
#### In [9]:

Change `ElemwiseCategorical(var=category` to `ElemwiseCategorical(vars=[category]`.
#### Current Cell Code:

``` python
# fit model
with model:
    step1 = pm.Metropolis(vars=[p, sd, means])
    step2 = pm.ElemwiseCategoricalStep(var=category, values=[0, 1, 2])
    tr = pm.sample(10000, step=[step1, step2])
```
#### Here is the updated Cell Code:

``` python
# fit model
with model:
    step1 = pm.Metropolis(vars=[p, sd, means])
    step2 = pm.ElemwiseCategoricalStep(vars=[category], values=[0, 1, 2])
    tr = pm.sample(10000, step=[step1, step2])
```

**Comment by twiecki** [comment]
Thanks for the report @x2apps, want to do a PR that fixes it?

**Comment by twiecki** [comment]
Closed with #958, thanks @x2apps
